ego/premain/core/core_test.go

```diff
@@ -200,8 +200,18 @@ func TestAddEnvVars(t *testing.T) {
 		Env:             []config.EnvVar{{Name: "HELLO_WORLD", Value: "2"}, {Name: "PWD", Value: "/tmp/somedir", FromHost: true}, {Name: "NOT_EXISTING_ON_HOST", FromHost: true}, {Name: "NOT_EXISTING_ON_HOST_BUT_INITIALIZED", Value: "42", FromHost: true}},
 	}
 
+	// Create key-value map from OS environment, as expected by addEnvVars
```

Comment:
In a unit test you shouldn't set them in the OS env in the first place, but just create the map as test input.